### ISSUE TITLE
https://cdn.auth0.com/js/auth0/9.10.4/auth0.js is serving wrong version

### ISSUE THREAD

**Issue by Pyrolistical** [auth0/auth0.js#956]
I was extremely confused with https://github.com/auth0/auth0.js/issues/953 because this bug was introduced in v9.11.0 (https://github.com/auth0/auth0.js/pull/952), but I was using cdn version of v9.10.4

So I downloaded and compared npm version of auth0-js@9.10.4 against https://cdn.auth0.com/js/auth0/9.10.4/auth0.js and they are different.

It seems like https://cdn.auth0.com/js/auth0/9.10.4/auth0.js is actually serving v9.11.0

**Comment by luisrudge** [comment]
Thanks for pointing this out. That's a very strange situation. For some reason, one of our builds passed through our checks and deployed to the cdn over another release:

```
Checking if file at https://cdn.auth0.com/js/auth0/9.10.4/auth0.min.js exists
File auth0.min.js exists for version 9.10.4
Uploading version 9.10.4
✔︎ auth0-js (9.10.4) uploaded to cdn
```

In the next build, it acted like it was supposed to:

```
Checking if file at https://cdn.auth0.com/js/auth0/9.10.4/auth0.min.js exists
File auth0.min.js exists for version 9.10.4
Skipping upload
```

I'll ping the infra team to understand what happened with that build and try to fix the cdn version.

**Comment by luisrudge** [comment]
Thanks for bringing this up. We fixed the cdn uploader for future releases in https://github.com/auth0/auth0.js/pull/958. I also fixed the cdn file hosted in the 9.10.4 version to have the correct release. 🎉
